### ISSUE TITLE
Ads Getting Through

### ISSUE THREAD

**Issue by ghost** [kaishin/Nope#19]
![ads](https://cloud.githubusercontent.com/assets/14188132/13195374/7bbe5880-d77e-11e5-8b6f-152f67863f7c.jpg)

**Comment by kaishin** [comment]
Cannot replicate this. Are you sure the website is not whitelisted?
![screenshot-2016-02-20](https://cloud.githubusercontent.com/assets/519433/13196303/bf1a13de-d7ca-11e5-9ee1-0e376bb20539.png)

**Comment by ghost** [comment]
Yeah I'm sure, it's really strange. I just did it again and all of those ads popped up.. :/

Edit: I tested with Ghosterly and it blocked all of them.

**Comment by kaishin** [comment]
Until I can replicate this I'm afraid I can't fix it. Will leave this open if someone has the same issue.

Please note that the test won't work anyway currently with ads blocked. I will look into that.

**Comment by vasuvasu** [comment]
Just installed Nope, and I'm having this issue too on beta.speedtest.net

**Comment by kaishin** [comment]
@vasuvasu What version are you using? I still can't replicate this... See in the screenshot what's being blocked and compare to what you see in your browser.
<img width="1432" alt="screenshot-2016-03-31" src="https://cloud.githubusercontent.com/assets/519433/14171289/67fbc6e2-f732-11e5-8f2d-924f5249ff61.png">

**Comment by ghost** [comment]
I'm guessing it's regional, different ad servers are used depending on where you are located / the IP is located.

**Comment by kaishin** [comment]
@SpacePuppeh Oh, that would make a lot of sense.

**Comment by ghost** [comment]
Are you using 40-50k rules? If not, adding more block lists and deduplicating them may help prevent things like this, or would any unused/rarely used rules make it slower?
